### PR TITLE
Homeworks 1-9 Tasks

### PR DESCRIPTION
**Pull request by Anton-web13**

### CODE REVIEW

File: js/two_block.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
+
+    if(value === 'Mark') {
+        console.log(`Hi, ${value}`)
+    } else {
+        console.log(`Hello, ${value}`)
+    }
+}
```
Comment (Boobaley) [inline js/two_block.js:9]: your function must return value, not just log

File: js/two_block.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
+
+    if(value === 'Mark') {
```
Comment (Boobaley) [inline js/two_block.js:4]: What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue

File: js/two_block.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
+//  1)
+const helloOleg = function(value) {
```
Comment (Boobaley) [inline js/two_block.js:2]: naming issue. Why Oleg?

File: js/two_block.js [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+helloOleg('Pol');
+
+
+//  2)
+const hypotenuse = function(n, m) {
+
+    const result = (n*n) + (m*m);
+    console.log(Math.sqrt(result));
+}
```
Comment (Boobaley) [inline js/two_block.js:19]: add return statements to your functions

File: js/two_block.js [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+
+hypotenuse(4, 4);
+
+
+// 3)
+const min = function(a ,b) {
+    let minValue = 0;
+    let maxValue = 0;
```
Comment (Boobaley) [inline js/two_block.js:27]: what the purpose of creating 2 variable ?

File: js/two_block.js [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+// 4 )
+const isEven = function(value) {
+    if(value % 2 === 0) {
+        value = 'true';
+    } else {
+        value = 'false';
+    }
+
+    console.log(value);
+}
```
Comment (Boobaley) [inline js/two_block.js:50]: Your function must return bool

File: js/two_block.js [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+
+isEven(98);
+
+// 5)
+const deleteChars = function() {
```
Comment (Boobaley) [inline js/two_block.js:55]: your function must apply arg not hardcoded hello

File: js/two_block.js [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+    let lastLetter = value.slice(value.length -1);
+
+    if(value.length % 2 === 0) {
+        lastLetter = lastLetter.toUpperCase();
+        console.log(`${resultOneLetter}${resultAllLetterWhitoutLastLetter}${lastLetter}`);
+    } else {
+        console.log(`${resultOneLetter}${resultAllLetter}`);
+    }
+
+}
```
Comment (Boobaley) [inline js/two_block.js:76]: add return statements to all your fns.

File: js/two_block.js [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+
+deleteChars();
+
+// 6)
+const stringValue = function(value) {
+    const resultOneLetter = value[0].toUpperCase();
+    const resultAllLetter = value.slice(1).toLowerCase();
+    const resultAllLetterWhitoutLastLetter = value.slice(1, -1).toLowerCase();
+    let lastLetter = value.slice(value.length -1);
```
Comment (Boobaley) [inline js/two_block.js:67]: 66,67 lines must be only if chars amount is even. no effort to calculate this every single time

File: js/two_block.js [hunk 10]
```diff
@@ -0,0 +1,118 @@
...
+// 9)
+const checkStringLength = function(string, number) {
+    if(string.length > number) {
+        console.log('String is too long!');
+    } else if (string.length < number) {
+        console.log('String is too little!');
+    } else {
+        console.log(string);
+    }
+}
```
Comment (Boobaley) [inline js/two_block.js:108]: read task attentively

**Review by Boobaley — CHANGES_REQUESTED** [review]

File: js/two_block.js (hunk 10 above)
Comment (Anton-web13) [reply js/two_block.js:108]: Corrected

File: js/two_block.js (hunk 9 above)
Comment (Anton-web13) [reply js/two_block.js:67]: Corrected

File: js/two_block.js (hunk 8 above)
Comment (Anton-web13) [reply js/two_block.js:76]: Corrected

File: js/two_block.js (hunk 7 above)
Comment (Anton-web13) [reply js/two_block.js:55]: Corrected

File: js/two_block.js (hunk 6 above)
Comment (Anton-web13) [reply js/two_block.js:50]: Corrected

File: js/two_block.js (hunk 5 above)
Comment (Anton-web13) [reply js/two_block.js:27]: Corrected

File: js/two_block.js (hunk 4 above)
Comment (Anton-web13) [reply js/two_block.js:19]: Corrected

File: js/two_block.js (hunk 3 above)
Comment (Anton-web13) [reply js/two_block.js:2]: Corrected

File: js/two_block.js (hunk 2 above)
Comment (Anton-web13) [reply js/two_block.js:4]: Corrected

File: js/two_block.js (hunk 1 above)
Comment (Anton-web13) [reply js/two_block.js:9]: Corrected

**Pull request merged**